app/assets/stylesheets/partials/_content_page.scss

```diff
@@ -0,0 +1,30 @@
+.container {
+  margin: 0 auto;
+  .content {
+    margin-left: 40em;
+    margin-right: 16em;
```

Comment:
I still think we'll want this as percentages because this will be a rigid layout otherwise, but we can debate that later.